public/src/__tests__/updateEnrollment.test.tsx

```diff
@@ -8,29 +8,47 @@ import { Route, Router } from "react-router"
 import { Provider } from "overmind-react"
 import { render, screen } from "@testing-library/react"
 import { MockData } from "./mock_data/mockData"
-import { Void } from "../../proto/qf/requests_pb"
+import { VoidSchema } from "../../proto/qf/requests_pb"
 import { initializeOvermind, mock } from "./TestHelpers"
 import { ApiClient } from "../overmind/effects"
-import { Timestamp } from "@bufbuild/protobuf"
-import { ConnectError } from "@bufbuild/connect"
+import { create } from "@bufbuild/protobuf"
+import { timestampFromDate } from "@bufbuild/protobuf/wkt"
+import { ConnectError } from "@connectrpc/connect"
 
 
 describe("UpdateEnrollment", () => {
     const api = new ApiClient()
     api.client = {
         ...api.client,
+        getEnrollments: mock("getEnrollments", async (request) => {
+            const enrollments: Enrollment[] = []
+            MockData.mockedEnrollments().enrollments.forEach(e => {
+                if (request.FetchMode?.case === "courseID") {
```

Comment:
Two questions.
1. Can this be expressed more compactly with a switch case?
2. Are there other cases than the ones covered? Because all cases does the same: `enrollments.push(e)`.

Reply:
Even better, it can be removed altogether!

We used to have to do `getCourse` and `getEnrollments` separately.
Following #1160 this is no longer the case, and only `getCourse` needs to be mocked here.